src/EditorFeatures/Core/Implementation/IntelliSense/AsyncCompletion/CompletionItemFilter.cs

```diff
@@ -0,0 +1,102 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Immutable;
+using Microsoft.CodeAnalysis.Editor.Shared.Extensions;
+using Microsoft.CodeAnalysis.PooledObjects;
+using Microsoft.CodeAnalysis.Tags;
+using Microsoft.VisualStudio.Core.Imaging;
+using Microsoft.VisualStudio.Language.Intellisense.AsyncCompletion.Data;
+using Microsoft.VisualStudio.Text.Adornments;
+using RoslynCompletionItem = Microsoft.CodeAnalysis.Completion.CompletionItem;
+
+namespace Microsoft.CodeAnalysis.Editor.Implementation.IntelliSense.AsyncCompletion
+{
+    internal static class CompletionItemFilter
```

Comment:
😕 I was expecting this to be removed

Also, do you know why it didn't show up as a rename in source control? Can you trying removing all unnecessary edits relative to the original and amend the commit so Git detects it as a rename instead of unrelated files?